src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1013,16 +1072,43 @@ private void BuildErrorBubble()
             ErrorBubble.ZIndex = ZIndex + 2;
 
             // The Node displays a count of dismissed messages, listening to that collection in the node's ErrorBubble
-            ErrorBubble.DismissedMessages.CollectionChanged += DismissedNodeWarnings_CollectionChanged;
+            
+            ErrorBubble.DismissedMessages.CollectionChanged += DismissedNodeMessages_CollectionChanged;
+        }
+
+        /// <summary>
+        /// Sets the color of the warning bar, which informs the user that the node is in
+        /// either an error or a warning state.
+        /// </summary>
+        /// <param name="style"></param>
+        /// <returns></returns>
+        private SolidColorBrush GetWarningColor(InfoBubbleViewModel.Style style)
+        {
+            switch (style)
+            {
+                case InfoBubbleViewModel.Style.None:
+                    break;
+                case InfoBubbleViewModel.Style.Warning:
+                case InfoBubbleViewModel.Style.WarningCondensed:
+                    return (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["NodeWarningColor"];
```

Comment:
I think it would be good if these strings were pulled out in to a private string so they only need to be updated in one place.